### PR TITLE
defuck

### PR DESCRIPTION
**Pull request by AutisticAndAshamed**

### CODE REVIEW

File: Flight Software/QT RP2040/Velocity data/code.py [hunk 1]
```diff
@@ -34,13 +32,13 @@
...
 time_0 = 0
-accel_0 = 0
+accel_0 = g
 
 # Ascent mode
 while True: #accel_mag > 30 or v0 > 10:
     icm_accel = icm.acceleration
     current_time = time.monotonic()
-    time_stamp = current_time - initial_time
+    time_1 = current_time - initial_time
```
Comment (besser435) [inline Flight Software/QT RP2040/Velocity data/code.py:41]: Should probably stay as time_stamp, makes it more clear what is or something idkr

File: Flight Software/QT RP2040/Velocity data/code.py [hunk 2]
```diff
@@ -51,12 +49,13 @@
...
         accel_sign = 1
-    vel_1 = accel_0 + (accel_sign * accel_mag - g - accel_0) * (time_stamp - time_0) * 0.5
-    time_0 += time_stamp
-    vel_0 += vel_1
-    accel_0 += (accel_mag * accel_sign)
+    accel_1 = accel_mag * accel_sign
+    vel_1 = vel_0 + (accel_1 + accel_0 - 2 * g) * (time_1 - time_0) * 0.5
+    time_0 = time_1
+    vel_0 = vel_1
+    accel_0 = accel_1
```
Comment (besser435) [inline Flight Software/QT RP2040/Velocity data/code.py:56]: Why are you setting variables to other variables, why not just reference the original?

**Review by besser435 — COMMENTED** [review]
Looks good r

File: Flight Software/QT RP2040/Velocity data/code.py (hunk 2 above)
Comment (AutisticAndAshamed) [reply Flight Software/QT RP2040/Velocity data/code.py:56]: It'll still work. I'll change it once the code is proven.

**Comment by AutisticAndAshamed** [comment]
This should probably be v0.2 since it's still in the testing phase

**Comment by AutisticAndAshamed** [comment]
It looks like the program works pretty well. The next step is to test it on an actual rocket, so I'll ask someone in the RDL if they want to fly it as a payload, and I can get FS^2 set up for some flights so we can compare the results with proven flight computers.

The biggest foreseeable problems we might encounter on a flight are drift (due to the assumptions made in the calculations) and the STEMMA QT connector being fucky during the flight. The first one could possibly be solved with a calibration system for all 3 axes, and the second one could be solved by soldering wires between the QT Py and accelerometer. Hopefully it shouldn't be necessary for short duration/average acceleration flights, but it will definitely be necessary to prevent another Tax Fraud situation.

**Review by besser435 — APPROVED** [review]

**Pull request merged**